solana-programs/anchor/audius-data/yarn.lock

```diff
@@ -315,6 +315,55 @@
     bn.js "^5.1.2"
     buffer-layout "^1.2.0"
 
+"@rollup/plugin-commonjs@^21.0.2":
```

Comment:
is there a change in dependencies?

Reply:
it probably changed earlier but my recent re-install picked up changed deps? i can remove yarn.lock and `yarn` from the install in CI; we should be relying on package-lock.json only.

Reply:
anchor has yarn as a direct dependency - i am a little unclear here but does package-lock.json always change when yarn.lock changes? https://book.anchor-lang.com/chapter_2/installation.html

Reply:
i think we should probably stick w/recommended installation flow to keep things simple

Reply:
ok. we should be consolidating around one - got this when running yarn. if this is the case then we should be deleting package-lock  
```
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
```

Reply:
works for me, let's stick to the recommended package manager for anchor (not sure why they use yarn tbh but just for consistency)

Reply:
ok! i consolidated around yarn; i had a package-json in my local but that was never committed to master. now that all installs are through yarn we should be ok here